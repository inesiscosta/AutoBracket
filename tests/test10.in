50 100
19 44 39 13 15 31 1 45 14 32 5 25 14 8 27 47 14 49 25 41 21 32 25 39 9 33 38 21 1 5 50 11 20 26 13 46 16 34 3 18 1 4 5 11 8 19 45 6 48 22
50 41 34 49 2 40 23 12 14 21 13 27 2 7 4 43 38 14 5 48 22 35 3 12 29 10 42 6 14 18 34 23 21 46 14 33 45 27 23 36 33 10 19 10 25 18 12 10 19 44
9 32 3 48 9 44 15 14 27 13 14 40 15 20 22 2 7 32 36 27 40 47 18 11 24 31 20 29 32 15 2 48 15 24 13 46 44 35 12 42 15 23 6 42 42 10 5 15 38 47
22 50 46 34 5 23 35 21 40 7 9 26 11 31 3 44 26 46 4 17 28 29 13 42 34 25 13 18 44 21 5 11 47 34 34 48 20 47 22 12 45 8 2 35 47 33 40 50 1 36
32 31 49 13 17 50 10 37 23 22 19 8 23 29 42 42 37 1 1 38 14 6 5 19 17 46 30 1 12 40 11 13 3 38 9 48 41 41 38 5 7 22 47 50 28 38 10 1 40 23
25 28 37 49 4 3 14 42 20 21 32 9 1 23 50 36 8 14 45 50 4 28 48 23 26 50 19 36 34 15 39 17 5 44 48 45 39 23 32 35 33 47 40 36 18 37 41 13 27 23
29 42 46 47 36 14 1 34 48 2 29 15 22 44 15 25 8 50 31 43 2 30 10 24 39 32 13 46 37 41 8 31 28 27 24 31 5 5 42 3 23 41 47 30 27 16 11 26 32 30
36 36 23 39 48 28 24 30 7 49 6 4 3 48 38 38 20 15 18 19 21 27 17 20 28 10 7 31 35 50 13 1 21 6 17 46 1 33 2 6 32 20 6 42 28 20 27 1 50 30
27 41 1 9 10 41 4 1 11 25 49 48 32 9 19 21 5 46 9 34 20 42 30 43 5 15 45 22 4 24 8 16 16 17 18 26 50 9 36 7 1 16 20 33 7 24 8 16 42 24
33 40 22 8 49 46 18 28 5 44 28 18 15 9 36 2 37 9 28 11 3 2 1 44 33 34 37 44 6 26 41 2 24 10 34 40 45 24 16 18 13 21 43 7 35 26 38 44 50 20
37 25 19 35 3 30 46 2 16 35 24 10 38 43 42 6 50 42 50 40 3 25 29 42 21 47 28 43 25 14 14 24 25 24 44 19 26 24 24 23 43 23 21 25 34 24 33 16 40 3
38 2 11 32 25 4 29 36 19 22 27 1 33 2 13 5 48 2 48 29 39 3 32 13 6 42 34 20 26 44 47 2 24 50 5 44 19 48 43 32 40 36 31 35 41 17 9 20 28 4
41 41 11 10 46 16 12 13 11 46 46 23 42 45 2 10 1 37 41 46 7 49 30 1 1 34 21 28 45 11 47 29 22 27 44 1 47 39 32 29 26 36 39 36 16 7 23 9 10 5
42 49 33 8 26 40 33 42 46 41 32 50 38 31 41 43 46 45 1 22 26 24 15 14 36 14 16 7 14 16 24 31 6 44 8 42 50 42 41 20 38 11 29 19 42 50 6 6 17 34
21 25 48 7 49 46 43 36 38 33 11 29 17 25 42 23 24 9 32 9 6 48 16 34 40 20 5 11 50 19 9 43 49 41 13 11 31 21 8 22 8 13 6 34 33 39 20 10 9 32
31 49 8 12 18 2 16 25 29 13 30 49 16 21 21 49 28 14 45 10 4 10 1 5 47 11 14 12 19 40 48 45 19 17 29 35 41 19 50 17 30 36 50 11 2 38 15 43 28 20
47 44 33 10 38 25 1 42 29 13 36 35 26 6 2 19 13 11 1 3 15 15 3 49 11 50 6 48 9 37 24 2 15 4 2 3 48 2 6 36 10 31 12 47 48 30 22 30 6 22
29 48 49 15 43 24 23 18 15 44 15 12 26 7 50 35 46 12 8 49 49 26 14 40 32 35 33 40 19 13 29 28 22 2 13 4 20 38 1 35 36 10 44 33 48 34 4 16 28 47
44 6 19 1 39 47 45 23 13 5 35 46 29 1 9 21 19 4 1 48 26 2 28 18 16 40 13 6 29 28 19 31 40 26 46 36 43 32 49 24 6 46 42 12 11 3 20 14 50 31
8 28 44 16 4 7 28 49 23 44 18 32 16 36 38 8 24 43 43 6 6 6 35 44 41 26 38 16 36 4 12 43 5 42 8 35 41 7 45 16 6 17 5 32 25 12 50 16 30 50
18 41 6 26 22 46 3 36 46 49 39 18 41 13 18 17 11 21 20 45 45 49 44 40 5 11 8 19 8 45 6 44 14 42 31 19 31 37 18 45 1 46 31 1 3 44 34 16 41 27
39 47 35 48 25 31 34 24 13 13 9 6 20 13 10 39 8 44 47 40 2 30 1 31 13 37 3 15 33 38 5 31 28 9 28 9 23 9 38 41 9 40 44 40 47 6 40 20 3 47
8 38 36 9 17 10 28 19 40 45 48 16 12 38 48 27 40 37 41 30 32 9 15 1 26 4 35 49 1 1 19 33 37 19 25 3 48 9 4 48 48 2 48 7 24 18 40 4 29 13
11 24 19 33 15 14 6 18 7 17 20 37 2 4 22 44 3 50 41 33 14 17 12 15 11 33 10 30 2 44 24 30 5 9 24 40 31 39 43 39 36 7 16 31 19 22 20 45 18 27
13 11 41 7 37 44 27 12 25 16 14 21 21 45 27 41 20 29 4 34 6 20 50 25 14 46 24 41 9 42 7 24 30 46 47 35 23 35 43 5 9 43 26 9 1 45 9 34 46 50
6 40 32 14 15 13 17 27 40 3 42 35 23 34 9 18 20 45 24 47 48 19 7 14 47 30 38 30 10 37 34 42 14 31 19 14 14 33 5 24 37 20 50 32 28 36 19 29 4 14
36 7 29 48 44 17 6 46 12 46 17 34 25 5 4 50 47 7 49 4 23 41 26 17 18 40 13 27 49 23 1 17 24 1 21 27 43 37 49 36 23 6 41 9 1 44 4 42 18 17
48 35 30 43 43 47 43 23 9 16 47 44 50 1 14 47 25 6 24 1 8 27 24 15 33 15 38 49 12 35 32 27 24 48 33 46 23 37 30 13 14 23 26 9 14 23 11 9 45 47
24 48 27 45 49 11 35 5 7 21 7 50 2 41 46 13 20 42 24 49 35 42 36 9 47 20 14 21 19 3 48 2 34 40 36 11 17 50 25 50 22 37 19 5 1 48 29 34 48 13
33 23 6 4 12 19 12 11 46 27 20 32 12 25 22 36 50 41 40 50 19 41 31 26 17 40 40 49 26 1 23 44 32 13 21 13 49 49 46 8 25 27 35 39 43 36 12 21 39 43
46 17 22 37 2 40 18 2 45 14 2 34 2 6 43 13 11 35 23 25 16 42 18 38 30 5 4 45 11 43 47 48 45 10 45 10 19 1 43 6 5 15 10 28 24 18 30 39 28 12
47 45 9 30 12 5 7 28 30 34 14 10 16 37 49 41 19 9 17 8 26 49 23 22 13 49 40 32 47 24 50 48 40 49 33 41 48 29 13 1 31 7 34 23 42 29 27 2 20 27
7 14 30 20 29 20 3 32 33 25 47 40 2 40 38 48 26 21 23 17 50 38 18 46 24 41 17 28 48 30 4 35 29 24 13 33 41 49 46 2 34 8 35 23 18 37 7 26 11 36
2 37 32 21 30 34 20 41 2 24 5 40 43 32 45 12 49 12 33 4 28 46 5 12 8 46 29 5 32 48 14 12 46 24 38 9 1 28 41 49 32 36 24 42 13 42 14 30 8 19
6 33 7 16 44 2 31 38 40 43 5 17 12 39 25 2 24 22 13 41 16 46 33 11 21 1 2 15 50 13 40 27 8 49 26 33 45 22 50 45 34 6 44 27 39 21 10 28 48 6
29 34 44 42 11 40 33 40 46 14 5 42 13 18 24 20 42 12 37 7 32 15 13 7 8 23 26 22 50 14 29 28 2 23 49 40 22 8 17 9 37 35 24 50 18 30 20 32 46 3
17 40 27 31 6 18 38 38 37 17 7 15 6 50 41 43 28 37 44 4 7 5 34 28 7 27 46 12 26 25 43 14 17 50 48 29 39 32 40 36 26 38 35 30 18 32 17 9 46 25
16 42 28 21 20 45 5 24 48 50 2 32 49 49 38 18 38 22 3 47 27 18 13 49 36 3 12 36 46 14 36 10 39 45 45 36 45 47 30 38 19 39 33 36 40 50 40 50 32 6
34 17 23 12 28 14 48 9 21 29 29 19 11 46 26 23 48 46 29 32 48 32 50 28 38 27 4 14 42 31 15 5 29 32 19 16 26 25 19 32 37 44 10 25 9 47 35 25 20 39
19 49 24 33 24 47 25 34 23 13 50 32 50 18 28 30 28 10 44 30 5 14 4 2 29 26 44 16 30 16 49 22 9 11 45 22 10 24 18 44 31 7 25 40 33 7 11 35 15 20
50 44 7 18 32 30 1 43 33 3 32 14 32 30 6 33 31 18 11 25 28 27 6 23 32 13 12 19 48 18 11 44 31 30 39 16 45 14 43 43 48 15 39 13 37 50 4 23 47 49
9 28 44 39 50 17 38 11 42 1 15 29 41 20 10 6 47 18 9 4 16 7 5 36 41 1 34 4 25 44 34 4 25 36 39 33 39 41 26 25 34 22 9 36 42 34 22 15 23 9
20 22 35 9 47 40 5 29 9 15 2 3 31 7 24 42 23 49 2 6 22 25 37 33 2 2 22 25 44 37 28 20 13 23 10 18 22 19 37 47 8 20 37 42 45 19 8 9 42 33
26 14 44 11 38 16 2 41 10 39 46 13 4 17 15 34 3 14 30 39 5 14 28 2 47 3 5 45 33 16 25 37 11 45 14 22 44 36 7 14 48 42 2 37 42 23 5 8 8 23
39 30 41 33 6 27 27 30 4 38 31 3 49 31 19 31 5 33 47 2 13 37 46 23 47 46 27 6 47 11 21 19 38 26 28 23 9 19 28 12 43 15 4 19 13 38 21 44 6 8
30 50 8 29 4 37 47 40 42 25 27 50 29 44 5 49 40 49 11 41 5 24 24 12 49 33 48 43 15 36 39 32 6 17 28 8 17 35 32 34 35 9 9 19 17 28 36 4 45 44
5 9 34 26 16 3 22 48 42 46 20 48 5 32 22 18 5 49 5 29 31 45 29 12 18 49 22 12 19 4 40 3 33 36 43 42 1 16 21 50 50 40 32 49 25 38 35 16 33 47
30 18 1 7 49 36 43 5 41 7 30 3 28 37 9 49 5 44 10 10 26 44 33 38 1 7 6 8 43 18 22 41 6 47 42 1 48 38 10 25 45 25 2 28 22 30 33 36 1 27
47 41 2 13 26 45 44 4 21 17 11 28 16 49 44 26 16 18 36 13 4 27 14 27 38 7 26 47 28 21 40 44 48 17 32 43 11 22 11 48 44 20 49 17 35 47 49 32 31 30
48 48 38 42 50 11 42 7 7 35 48 29 4 42 30 45 32 50 22 46 38 47 48 40 27 6 49 1 29 7 50 37 40 19 45 29 23 39 38 25 37 32 49 17 20 33 30 31 6 34
3 4 12 24 10 35 29 49 16 42 19 14 35 12 7 30 34 2 14 48 14 23 4 20 38 35 41 3 2 28 11 49 13 34 32 44 43 50 26 25 45 44 44 33 13 34 24 46 31 26 34 33 46 46 13 15 14 30 42 40 46 14 26 21 47 3 26 2 33 19 1 7 41 29 36 26 47 48 34 40 6 7 28 23 44 33 45 9 7 15 20 39 17 24 19 22 9 31 34 14
38
