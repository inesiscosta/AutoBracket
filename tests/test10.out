1
((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((3 4) 12) 24) 10) 35) 29) 49) 16) 42) 19) 14) 35) 12) 7) 30) 34) 2) 14) 48) 14) 23) 4) 20) 38) 35) 41) 3) 2) 28) 11) 49) 13) 34) 32) 44) 43) 50) 26) 25) 45) 44) 44) 33) 13) 34) 24) 46) 31) 26) 34) 33) 46) (46 13)) 15) 14) (30 42)) 40) 46) 14) (26 21)) 47) 3) 26) 2) 33) 19) 1) 7) 41) 29) 36) 26) 47) 48) 34) 40) (6 7)) 28) (23 44)) 33) 45) 9) 7) 15) 20) 39) 17) 24) 19) 22) 9) 31) 34) 14)
